src/uv-common.c

```diff
@@ -155,6 +155,82 @@ static const char* uv__unknown_err_code(int err) {
   return copy != NULL ? copy : "Unknown system error";
 }
 
+int uv_translate_posix_error(int sys_errno) {
+  switch (sys_errno) {
```

Comment:
This is what I'm planning to change:

- Drop the names `uv_translate_posix_error` and `uv_translate_win32_error`.
- Expose `uv_translate_sys_error`
  - Unix implementation: negated errno
  - Windows implementation: the same as the current one

Does this sound right?